_stylesheet.css

```diff
@@ -1,6 +1,34 @@
 /* Main Layout */
+html {
+    position: relative;
```

Comment:
Ich sehe da kein Problem drin, wenn man ein Design an webSPELL anpasst. Braucht man doch nur alle CSS Deklarationen die in den Menüpunkt /* Main Layout */ fallen rauslöschen. Das wird jeder gescheite Anpasser so handhaben, bzw. sollte es so handhaben. Dann macht es auch keinen unterschied mehr ob dort mit position: relative; gearbeitet wird oder nicht.